api/controllers/game/subscribe.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (req, res) {
         } else {
           pNum = (game.players[0].pNum + 1) % 2;
           await Game.updateOne({ id: game.id }).set({
-            status: false,
+            status: gameService.GameStatus.CREATED,
```

Comment:
Is this needed considering this should be set when the game created and then not changed until both players are ready? Was it here as a safety?